packages/apm/src/integrations/tracing.ts

```diff
@@ -389,7 +412,10 @@ export class Tracing implements Integration {
    * Removes activity and finishes the span in case there is one
    */
   public static popActivity(id: number, spanData?: { [key: string]: any }): void {
-    if (!Tracing._isEnabled()) {
+    // The !id is on purpose to also fail with 0
+    // Since 0 is returned by push activity in case tracing is not enabled
```

Comment:
Isn't this case already covered by `!Tracing._isEnabled()`?

Reply:
@rhcarvalho there are cases when `popActivity()` is called within a `setTimeout()`